.github/workflows/docker.yml

```diff
@@ -37,14 +37,14 @@ jobs:
           else
             # For PR, use PR branch name
             if [[ ${{ github.event_name }} == 'pull_request' ]]; then
-              BRANCH=$GITHUB_HEAD_REF
+              PR_TAG="pr-${{ github.event.pull_request.number }}"
```

Comment:
calling this `PR_TAG` instead of `BRANCH` might be misleading because we don't populate just for PRs. Can we keep the original `BRANCH` variable name or just `TAG`?

Reply:
we would only reach this else block if it is a PR @fxenik

Reply:
Yeap but we're setting PR_TAG in the block after that as well, where the main branch is handled, right?

Reply:
ah got it, makes sense